src/desktop/apps/auction/components/App.js

```diff
@@ -13,15 +13,15 @@ export default class App extends Component {
 
   render() {
     return (
-      <Theme>
+      <StitchWrapper>
```

Comment:
[`StitchWrapper`](#) includes a relay provider, so this should allow mounting a modal that contains its own relay queries and actions. It also includes the ThemeProvider.

Reply:
`StitchWrapper` is specifically for stitch middleware functionality -- we should be using `<SystemContextProvider>` directly here